frontend/utils/index.ts

```diff
@@ -10,3 +11,7 @@ export function fromEntries<V>(iterable: Iterable<[string, V]>) {
     return obj
   }, {} as { [k: string]: V })
 }
+
+export const sanitize = (html: string): string => {
```

Comment:
Created this util function so that we can swap out the implementation with another library in just one place if we ever need to

Reply:
Nice! 🎉